imap_processing/hi/l1b/hi_l1b.py

```diff
@@ -56,14 +55,24 @@ def hi_l1b(l1a_cdf_path: Path, data_version: str):
             converters={"mnemonic": str.lower},
         )
 
-        l1b_dataset.attrs.update(hi_hk_l1b_global_attrs.output())
+        l1b_dataset.attrs.update(
+            CDF_MANAGER.get_global_attributes("imap_hi_l1b_hk_attrs")
+        )
     elif logical_source_parts[-1].endswith("de"):
         l1b_dataset = annotate_direct_events(l1a_dataset)
     else:
         raise NotImplementedError(
             f"No Hi L1B processing defined for file type: "
             f"{l1a_dataset.attrs['Logical_source']}"
         )
+    # Update global attributes
+    # TODO: write a function that extracts the sensor from Logical_source
+    #    some functionality can be found in imap_data_access.file_validation but
+    #    only works on full file names
+    sensor_str = logical_source_parts[-1].split("-")[0]
```

Comment:
Before you had like this, was that not working?
```
sensor_str = l1a_dataset.attrs["Logical_source"].split("_")[-1].split("-")[0]
```

Reply:
That was working, but it was parsing directly from `Logical_source`. I moved this code up into the `hi_l1b` function where I had already performed the first part of the parsing. Line 39 of `hi_l1b.py`:
`logical_source_parts = l1a_dataset.attrs["Logical_source"].split("_")`.

Ultimately, this logic should be moved to a common location so that any instrument that wants to parse the sensor from Logical_source or filename can call that function.

Reply:
Ah ok. Sorry, missed that part. Cool, that makes sense.